petab/measurements.py

```diff
@@ -47,6 +66,9 @@ def get_noise_distributions(measurement_df: pd.DataFrame) -> dict:
     Returns:
         Dictionary with `observableId` => `cost definition`
     """
+    warn("This function will be removed in future releases.",
```

Comment:
why not just remove?

Reply:
Still used in the script to update assignmentrule-style models. Would kick out in 0.2.0.